lib/tabs.sh

```diff
@@ -38,6 +42,20 @@ function start_sessions()
             sleep 0.1
             qdbus-qt5 $konsole_service /Sessions/$session_num sendText $'\n'
             sleep 0.1
+        elif [[ $TILIX_ID ]]; then
+            echo "Splitting Tilix window and executing command: $name"
+            xdotool key ctrl+alt+d # Split downwards
```

Comment:
This will probably not work on Wayland. I'd still merge this change; I'm just saying the usefulness is maybe limited.

Reply:
yes i porbably wouldnt merge in this state, i'm still looking for a better solution as although it does work for wayland currently, it does not recignize the keymap settings (it prints using the us layout, e.g. `openqa-start` as `openqaßstart`, afaik because it looks at the x11 settings instead of waylands)
I've had a look at ydotool and wl-clipboard but so far that did not work great either, i'll try again though

Reply:
Although this whole approach is kind of questionable. Probably it makes more sense to simply use a terminal that's scriptable like Konsole or tmux - even though it is not your favorite terminal for other usages. (I suppose you have checked whether Tilix has some kind of API, e.g. a D-Bus API? Note that it might not be documented but you can easily check it out using e.g. `qdbusviewer6`.)